package.json

```diff
@@ -44,6 +44,7 @@
     "karma-jasmine": "^1",
     "karma-phantomjs-launcher": "^1",
     "lasso-cli": "^2",
+    "mocha": "^4.0.1",
```

Comment:
Why the need for mocha?

Reply:
I added a server side test.

Reply:
This feels more like an integration test with the environment than a unit test. I'm not sure how much value we get out of it...? Again, just trying to keep the dependencies and npm scripts down. I will have to consider if this adds real value, and if so I would update all makeup modules to follow suit.

Reply:
not much, just to be sure it was working and hopefully lessen that coveralls warning, but I realized afterward coverage was only generated for FE tests, doh!